# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../grub/grub-0.91-splashimagehelp.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ur grub-0.91/docs/grub.texi grub/docs/grub.texi
--- grub-0.91/docs/grub.texi	Mon Jan 21 22:57:46 2002
+++ grub/docs/grub.texi	Mon Jan 21 22:57:51 2002
@@ -1891,6 +1891,7 @@
 * rarp::                        Initialize a network device via RARP
 * serial::                      Set up a serial device
 * setkey::                      Configure the key map
+* splashimage::                 Use a splash image
 * terminal::                    Choose a terminal
 * tftpserver::                  Specify a TFTP server
 * unhide::                      Unhide a partition
@@ -2260,6 +2261,16 @@
 @end deffn
 
 
+@node splashimage
+@subsection splashimage
+
+@deffn Command splashimage file
+Select an image to use as the background image.  This should be
+specified using normal GRUB device naming syntax.  The format of the
+file is a gzipped xpm which is 640x480 with a 14 color palette.
+@end deffn
+
+
 @node terminal
 @subsection terminal
 
